queries.yaml

```diff
@@ -432,6 +432,23 @@ post_process:
         - transit
         - water
 
+  # drop the names of small landuse polygons
+  - fn: vectordatasource.transform.drop_properties
+    params:
+      source_layer: landuse
+      start_zoom: 9
```

Comment:
We show a few kinds earlier as zoom 2 (https://github.com/tilezen/vector-datasource/blob/master/yaml/landuse.yaml#L5-L13).